documentation/help-documenting.rst

```diff
@@ -36,11 +36,11 @@ and :ref:`maintenance <maintbranch>` branches at https://docs.python.org/dev/.
 The in-development and recent maintenance branches are rebuilt once per day.
 
 If you would like to be more involved with documentation, consider subscribing
-to the `docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_
-mailing list and the `Documentation category on the Python Discourse
-<https://discuss.python.org/c/documentation/26>`_,
-where user issues are raised and documentation toolchain, projects, and standards
-are discussed.
+to the `Documentation category on the Python Discourse
+<https://discuss.python.org/c/documentation/26>`_ and the
+`docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_
+mailing list where user issues are raised and documentation toolchain,
+projects, and standards are discussed.
```

Comment:
```suggestion
`docs@python.org <https://mail.python.org/mailman3/lists/docs.python.org/>`_ mailing list
where user issues are raised and documentation toolchain, projects, and standards
are discussed.
```

Remaking my suggestion again to minimize unnecessary diff noise, as was generally agreed in #1018 — this avoids changing the latter two otherwise-untouched lines solely due to hard wrapping.